src/main/java/core/basesyntax/vehicles/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.vehicles;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Truck is working");
```

Comment:
The message in the `doWork` method should be more informative and indicate the specific type of machine that is working. According to the checklist, the message should be something like 'Truck started its work.'